src/doc/rustc/src/platform-support/wasm32-wasip1-threads.md

```diff
@@ -162,3 +162,17 @@ It's recommended to conditionally compile code for this target with:
 Prior to Rust 1.80 the `target_env = "p1"` key was not set. Currently the
 `target_feature = "atomics"` is Nightly-only. Note that the precise `#[cfg]`
 necessary to detect this target may change as the target becomes more stable.
+
+## Enabled WebAssembly features
+
+The default set of WebAssembly features enabled for compilation is similar to
+[`wasm32-unknown-unknokwn`](./wasm32-unknown-unknown.md) but two more features
+are included:
```

Comment:
"similar to `wasm32-unknown-unknown` but two more features are included" doesn't specify whether features originally included in `wasm32-unknown-unknown` are no longer included.

Reply:
Note that there is a typo in the second `unknokwn` at line 169.